### ISSUE TITLE
[Android] createNotificationChannel exception

### ISSUE THREAD

**Issue by archangelnl** [Baseflow/XamarinMediaManager#637]
## 🐛 Bug Report

So I've recently started integrating XamarinMediaManager into one of our apps for HLS playback, and I've looked over into the apps Android vitals recently as I noticed a lot of crashes were happening, and found that there's a lot of crashes relating to android.content.res.Resources$NotFoundException

This should obviously not be happening, and as such you can find a stacktrace below.

```
java.lang.RuntimeException: 
at android.app.ActivityThread.handleCreateService (ActivityThread.java:3981)
at android.app.ActivityThread.-wrap5 (Unknown Source)
at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2092)
at android.os.Handler.dispatchMessage (Handler.java:108)
at android.os.Looper.loop (Looper.java:166)
at android.app.ActivityThread.main (ActivityThread.java:7523)
at java.lang.reflect.Method.invoke (Native Method)
at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:245)
at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:921)

Caused by: android.content.res.Resources$NotFoundException: 
at android.content.res.Resources.getText (Resources.java:410)
at android.content.res.HwResources.getText (HwResources.java:465)
at android.content.res.Resources.getString (Resources.java:504)
at android.content.Context.getString (Context.java:560)
at com.google.android.exoplayer2.util.NotificationUtil.createNotificationChannel (Unknown Source:16)
at com.google.android.exoplayer2.ui.PlayerNotificationManager.createWithNotificationChannel (Unknown Source:1)
at md5bc84c97450f22337f6e1991a43223866.MediaBrowserService.n_onCreate (Native Method)
at md5bc84c97450f22337f6e1991a43223866.MediaBrowserService.onCreate (Unknown Source)
at android.app.ActivityThread.handleCreateService (ActivityThread.java:3961)
```

### Expected behavior

App should not be crashing due to a Resource not found exception

### Reproduction steps

```
item = await CrossMediaManager.Current.Extractor.CreateMediaItem("http://streaming1.stream.delivery/radiocontinu/live/playlist.m3u8");
item.MediaType = MediaType.Hls;
item.Artist = "Radio Continu";
item.Title = "Livestream";
item.Album = "Radio Continu";
this.VideoPlayer.Source = item;
```

Sadly, I have been unable to reproduce this myself, though I thought I should still log it as a bug report to see if you perhaps have any idea as to why this issue is taking place.

I have also tried setting CrossMediaManager.Android.NotificationIconResource to a different drawable but to no avail.

### Configuration

**Version:** 0.8.18

**Platform:** 
- [ ] :iphone: iOS
- [X] :robot: Android
- [ ] :checkered_flag: WPF
- [ ] :earth_americas: UWP
- [ ] :apple: MacOS
- [ ] :tv: tvOS
- [ ] :monkey: Xamarin.Forms

**Comment by MagnusEkroth** [comment]
My users have the same crashes in my android app. Unable to reproduce with my phones.

Look like it only affect users with Huawei phones with Android 8.0 and 8.1

Huawei P20 Pro (HWCLT) | 180 | 61.0%
HUAWEI Y6 2018 (HWATU-QG) | 115 | 39.0%

I have a Huawei with Android 9.0 and have no luck reproducing the crash there.

Using version 0.8.18 and Xamarin Forms 4.3

**Comment by AztecWorks** [comment]
My users are also having this problem. Like MagnusEkroth said it is only affecting HUAWEI phones
Huawei Mate 10 lite (HWRNE)
HUAWEI nova 2 Plus (HWBAC)
HUAWEI Y7 Prime 2018 (HWLDN-Q)

java.lang.RuntimeException: 
  at android.app.ActivityThread.handleCreateService (ActivityThread.java:3987)
  at android.app.ActivityThread.-wrap5 (Unknown Source)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2092)
  at android.os.Handler.dispatchMessage (Handler.java:108)
  at android.os.Looper.loop (Looper.java:166)
  at android.app.ActivityThread.main (ActivityThread.java:7529)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:245)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:921)
Caused by: android.content.res.Resources$NotFoundException: 
  at android.content.res.Resources.getText (Resources.java:410)
  at android.content.res.HwResources.getText (HwResources.java:465)
  at android.content.res.Resources.getString (Resources.java:504)
  at android.content.Context.getString (Context.java:560)
  at com.google.android.exoplayer2.util.NotificationUtil.createNotificationChannel (NotificationUtil.java:87)
  at com.google.android.exoplayer2.ui.PlayerNotificationManager.createWithNotificationChannel (PlayerNotificationManager.java:356)
  at md5bc84c97450f22337f6e1991a43223866.MediaBrowserService.n_onCreate (Native Method)
  at md5bc84c97450f22337f6e1991a43223866.MediaBrowserService.onCreate (MediaBrowserService.java:34)
  at android.app.ActivityThread.handleCreateService (ActivityThread.java:3967)

**Comment by OdetteP** [comment]
Made a PR so app will not crash anymore.

**Comment by martijn00** [comment]
Fixed in 0.9.1
